cms/envs/help_tokens.ini

```diff
@@ -12,12 +12,12 @@ updates = course_author:course_assets/handouts_updates.html
 pages = course_author:course_assets/pages.html
 files = course_author:course_assets/course_files.html
 textbooks = course_author:course_assets/textbooks.html
-schedule = course_author:set_up_course/setting_up_student_view.html
+schedule = course_author:set_up_course/studio_add_course_information/index.html
 grading = course_author:grading/index.html
-team_course = course_author:set_up_course/course_staffing.html#add-course-team-members
+team_course = course_author:set_up_course/studio_add_course_information/studio_course_staffing.html
 team_library = course_author:course_components/libraries.html#give-other-users-access-to-your-library
 advanced = course_author:index.html
-checklist = course_author:set_up_course/creating_new_course.html
+checklist = course_author:set_up_course/index.html
```

Comment:
I saw in the test files does not have a test for this link. As this is high priority, you can go ahead a merge this and discuss start a conversation regarding if this needs a test.